FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentMacro.cs

```diff
@@ -0,0 +1,38 @@
+using FFXIVClientStructs.FFXIV.Component.GUI;
+
+namespace FFXIVClientStructs.FFXIV.Client.UI.Agent;
+
+[Agent(AgentId.Macro)]
+[StructLayout(LayoutKind.Explicit, Size = 0xbf0)]
```

Comment:
the size of AgentMacro is `0xEB18`

Reply:
That'll teach me to trust Ghidra auto-fill-in-structure. I'll update this now :+1:

Reply:
Done